pkg/bgpv1/manager/reconcile.go

```diff
@@ -237,35 +237,43 @@ func neighborReconciler(
 	nset := map[string]*member{}
 
 	// populate set from universe of new neighbors
-	for i, n := range newNeigh {
+	for i := range newNeigh {
+		n := newNeigh[i]
+		// Set neighbor default port for peering if unspecified. This is done by kube-apiserver
+		// but added here for non-k8s environments, tests, etc.
+		n.SetDefaultPeerPort()
 		var (
-			key = fmt.Sprintf("%s%d", n.PeerAddress, n.PeerASN)
+			key = fmt.Sprintf("%s%d%d", n.PeerAddress, *n.PeerPort, n.PeerASN)
```

Comment:
Not sure why we need port number in the key. Each peer is uniquely identified by just the address. Gobgp checks uniqueness based on [address field](https://github.com/osrg/gobgp/blob/master/pkg/server/server.go#L2997 ). 

I think all the logs where we are printing out peer address:port combo can be omitted as well, since port is not providing any extra information.

Reply:
> Each peer is uniquely identified by just the address.

@harsimran-pabla thanks for the thorough review and feedback. If this is the case, is the ASN included to provide additional context?